### PR TITLE
Update the hyperlinks of api-conventions.md file in comments

### PR DESCRIPTION
**Pull request by niconical**
#### Which component this PR applies to?
addon-resizer,vertical-pod-autoscaler

#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
The link to api-conventions markdown file has been changed, but many of the code comments in the repo have not been updated, which can cause inconsistencies.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?
NONE

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by jbartosik — CHANGES_REQUESTED** [review]
Please don't change vendored files. Please don't make any changes to files in `addon-resiser/vendor` and `vertical-pod-autoscaler/e2e/vendor` directories (or any other vendor directories).

**Comment by niconical** [comment]
> Please don't change vendored files. Please don't make any changes to files in `addon-resiser/vendor` and `vertical-pod-autoscaler/e2e/vendor` directories (or any other vendor directories).

Thanks a lot for your suggestion, I didn't figure out what the vendor directory means before. The changes have now been resubmitted.

File: vertical-pod-autoscaler/deploy/vpa-v1-crd-gen.yaml [hunk 1]
```diff
@@ -14,196 +14,212 @@ spec:
     listKind: VerticalPodAutoscalerCheckpointList
     plural: verticalpodautoscalercheckpoints
     shortNames:
-    - vpacheckpoint
+      - vpacheckpoint
```
Comment (jbartosik) [inline vertical-pod-autoscaler/deploy/vpa-v1-crd-gen.yaml:17]: Please don't change formatting in this file.

**Review by jbartosik — CHANGES_REQUESTED** [review]
One more comment.

Mostly looks good now.

File: vertical-pod-autoscaler/deploy/vpa-v1-crd-gen.yaml (hunk 1 above)
Comment (niconical) [reply vertical-pod-autoscaler/deploy/vpa-v1-crd-gen.yaml]: Thanks, the formatting issue has been fixed.

**Comment by niconical** [comment]
Thanks for your time, I have fixed all known issues.

File: vertical-pod-autoscaler/deploy/vpa-v1-crd-gen.yaml (hunk 1 above)
Comment (niconical) [reply vertical-pod-autoscaler/deploy/vpa-v1-crd-gen.yaml]: > Please don't change formatting in this file.

Thanks, the formatting issue has been fixed.

**Review by jbartosik — APPROVED** [review]
/lgtm

/approve

**Comment by gjtempleton** [comment]
Thanks!
/approve

**Pull request merged**